tests/framework/PostProcessors/Validation/PhysicsGuidedCoverageMapping/pcm_metric_dump.csv

```diff
@@ -0,0 +1,2 @@
+pri_post_stdReduct_time900
```

Comment:
This file need to be moved to gold folder

Reply:
Output files have been moved to gold folder.